### PR TITLE
Introduce oguri.service

### PR DESCRIPTION
**Pull request by jtheoof**
Oguri is a nice wallpaper deamon for Wayland compositors with support
for static as well as animated wallpapers.

This commit introduces the first version of a systemd config.

See [oguri](https://github.com/vilhalmer/oguri)

### CODE REVIEW

**Comment by jtheoof** [comment]
Any chance to get this merged @xdbob ?

**Comment by markstos** [comment]
@jtheoof Why not ask the `oguri` project to ship their own systemd service files first? I checked their bug tracker and don't see any discussion about that there.

**Comment by xdbob** [comment]
At first glance the PR looks pretty good, I'm sorry I haven't given sway-services the love it deserves lately. Give me the weekend to try `oguri` before merging (don't hesitate to ping me next week if I don't reply next week).

Nitpick: I think you should remove the `Restart=always` from the unit, I know you took it from my units but I now think this was a mistake

**Comment by jtheoof** [comment]
@markstos Unfortunately, this doesn't work so well because this project uses a non standard `wayland-session.target`. If we put this unit service in `oguri` source code, it needs to be more "standard".

See for example what happened in:

- [waybar](https://github.com/Alexays/Waybar/pull/801/files)
- [mako](https://github.com/emersion/mako/commit/16fbced1f92efe6004349fe165cc97225ecccdc5)

Both projects are using (or have switched to) a more standard `graphical-session.target` but those don't play so well with `sway-service`. For example in the case of `waybar`, some modules are not initialized properly (im my case, either the waybar is loaded too early (before wayland is ready) or the workspace module doesn't load (IPC not ready))

Ideally, `sway-service` would target a more standard `graphical-session.target` with the use of [ConditionEnvironement](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#ConditionEnvironment=). (The same way mako does it). This would be a separate task with proper testing.

For the moment, since `oguri` is a rather small project, I think it's OK to put it this repo instead. We can always migrate to oguri later on.

@xdbob I've re-pushed with your recommendation.

File: systemd/oguri.service [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=A very nice animated wallpaper daemon for Wayland compositors
```
Comment (xdbob) [inline systemd/oguri.service]: Can you remove the value judgment "very nice" ? (I know it's from upstream but still)

Comment (jtheoof) [reply systemd/oguri.service]: lol, sure

Comment (jtheoof) [reply systemd/oguri.service]: Done

**Comment by xdbob** [comment]
Thanks

**Pull request merged**